instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/builder/internal/DefaultHttpServerInstrumenterBuilder.java

```diff
@@ -177,7 +185,7 @@ public Instrumenter<REQUEST, RESPONSE> build() {
     return builder.buildInstrumenter(SpanKindExtractor.alwaysServer());
   }
 
-  private InstrumenterBuilder<REQUEST, RESPONSE> builder() {
+  public InstrumenterBuilder<REQUEST, RESPONSE> builder() {
```

Comment:
I could not find any usages for this method outside of this class.

Reply:
added a `instrumenterBuilder` method instead and only exposed propagators of `OpenTelemetry`

Reply:
inlined